src/w3iProxy/w3iChatFacade.ts

```diff
@@ -35,6 +47,58 @@ class W3iChatFacade implements W3iChat {
 
     const ProviderClass = this.providerMap[this.providerName]
     this.provider = new ProviderClass(this.emitter)
+
+    // Discuss expiry of messages
```

Comment:
Context here? I.e. message expiry is unclear for retries specifically or beyond that too?